wgengine/wgengine.go

```diff
@@ -138,14 +138,14 @@ type Engine interface {
 	// the given IP and then call cb with its ping latency & method.
 	Ping(ip netaddr.IP, cb func(*ipnstate.PingResult))
 
-	// RegisterIPPortIdentity registers a given node as temporarily having
-	// the given IP:port for whois lookups.
-	RegisterIPPortIdentity(netaddr.IPPort, *tailcfg.Node)
+	// RegisterIPPortIdentity registers a given node (identified by its
+	// Tailscale IP) as temporarily having the given IP:port for whois lookups.
```

Comment:
elaborate a bit more here to clarify that the IP is almost always 127.0.0.1 (or some other localhost address) and the port is some ephemeral port for proxying to localhost.